kymatio/scattering3d/scattering3d.py

```diff
@@ -79,19 +77,46 @@ def build(self):
         self.gaussian_filters = gaussian_filter_bank(
                                 self.M, self.N, self.O, self.J + 1, self.sigma_0)
 
-    def cuda(self):
-        """Move to the GPU
+        # transfer the filters from numpy to torch
+        for k in range(len(self.filters)):
+            self.filters[k] = torch.from_numpy(self.filters[k]).type(torch.Tensor)
```

Comment:
why `.type(torch.Tensor)`? What does that do? How can the output of `torch.from_numpy(self.filters[k])` not be a `torch.Tensor` ?

Reply:
Ok, so the point is that by default, the type of the returned tensor(here desired float32) corresponds to the type of the numpy array(here np.64). On the other hand, I think it makes sens to use the default type set up in torch(eg torch.Tensor, which is normally float32). It will be helpful the day we will be float64 compatible!